src/extension/app/components/action-bar/activity-action/activity-action.css.js

```diff
@@ -45,6 +45,7 @@ export const style = css`
     align-items: center;
     flex-direction: row;
     width: 100%;
+    white-space: pre-wrap;
```

Comment:
adding the white-space:pre-wrap , so that it will wrap the long messages in the toast

Reply:
@kmurugulla Let's also add `flex-shrink: 0;` on the svg icon on the left.

Reply:
thank you and added the changes

Reply:
Just noticed this also caused the close icon to disappear on the right.

Reply:
sorry for not realizing that the close button disappeared. I moved the wrap behavior down to the span tag where the message is appearing which seems to apply the wrapping to the longer messages , show both info and close icons as they are supposed to be